packages/chat-component/src/components/chat-avatar.ts

```diff
@@ -0,0 +1,33 @@
+import { LitElement, html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { styles } from '../styles/chat-avatar.js';
+import { unsafeSVG } from 'lit/directives/unsafe-svg.js';
+
+export interface ChatStage {
+  label: string;
+  svgIcon: string;
+  url: string;
+}
+
+@customElement('chat-avatar')
+export class ChatAvatarComponent extends LitElement {
+  static override styles = [styles];
+
+  @property({ type: String })
+  label = '';
+
+  @property({ type: String })
+  svgIcon = '';
+
+  @property({ type: String })
+  url = '';
+
+  override render() {
+    return html`
+      <a class="chat-avatar__link" title="${this.label}" href="${this.url}" target="_blank" rel="noopener noreferrer">
```

Comment:
styles inside a component are isolated and will not be impacted by outside styling of <a> component. So you dont need a class and can style it using as a 'a' link.

Reply:
That's true. I did deal so much with specificity over the years I still struggle to target native elements for styling, even inside of shadow DOM. I still do believe it may pose extensibility challenges at that granular, encapsulated level. But for the sake of avoiding premature optimization, I updated.